src/services/EventService.js

```diff
@@ -2,7 +2,7 @@
 import axios from 'axios';
 
 const TOKEN_KEY = 'JWT';
-const URL = `http://34.67.193.145:${process.env.PORT || 5000}`;
+const URL = `http://localhost:${process.env.PORT || 5000}`;
```

Comment:
Maybe comment out the other one instead of removing it so it's easier to switch between localhost and server

Reply:
Will do :+1: